src/types/config.ts

```diff
@@ -40,6 +40,7 @@ export type ConfigV3 = {
     downloadLocation: string;
     spellCheckerURL?: string;
     lastActiveTeam?: number;
+    startInFullscreen: boolean;
```

Comment:
This should be optional.
```suggestion
    startInFullscreen?: boolean;
```

Reply:
Thanks! What makes it require the optional? I notice most aren't optional params. I assume this has to do with params added after the V3 model was released?

Reply:
That, for one thing, and also it makes sense for stuff like booleans to be optional since we can just check for a falsy value and have the same behaviour happen.